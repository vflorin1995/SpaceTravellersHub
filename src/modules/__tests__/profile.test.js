import renderer from 'react-test-renderer';
import { HashRouter as Router } from 'react-router-dom';
import { Provider } from 'react-redux';
import store from '../../redux/configureStore';
import MyProfile from '../myProfile';

describe('Jest Snapshot testing suite', () => {
  it('Matches DOM Snapshot for the profile component', () => {
    const quotes = renderer.create(
      <Provider store={store}>
        <Router>
          <MyProfile />
        </Router>
      </Provider>,
    ).toJSON();
    expect(quotes).toMatchSnapshot();
  });
});
