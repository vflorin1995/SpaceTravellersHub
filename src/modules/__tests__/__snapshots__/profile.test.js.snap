// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Jest Snapshot testing suite Matches DOM Snapshot for the profile component 1`] = `
<div
  className="grid grid-cols-2 m-8 gap-4"
>
  <div
    className="flex flex-col gap-4"
  >
    <table
      className="w-full"
    >
      <thead>
        <tr>
          <td
            className="text-2xl font-bold capitalize pb-4"
          >
            my missions
          </td>
        </tr>
      </thead>
      <tbody>
        <tr
          className="flex flex-col"
        >
          <td
            className="py-2 border px-1"
          >
            You do not have any rocket reservations yet
          </td>
        </tr>
      </tbody>
    </table>
  </div>
  <div>
    <table
      className="w-full"
    >
      <thead
        className="text-2xl mb-2"
      >
        <tr>
          <td
            className="text-2xl font-bold capitalize pb-4"
          >
            My rockets
          </td>
        </tr>
      </thead>
      <thead
        className="w-full"
      />
    </table>
  </div>
</div>
`;
